src/query/server/query.go

```diff
@@ -178,9 +178,6 @@ func Run(runOpts RunOptions) {
 
 	// Close metrics scope
 	defer func() {
-		if e := recover(); e != nil {
```

Comment:
What's the motivation for removing the recover here?

Reply:
Was dropping because I hit this case and wasn't able to see actual line that was panicing since error log just printed the error message text without a stack.  

As discussed, would prefer an actual panic here with the stack dump.